src/Phantomjs.coffee

```diff
@@ -12,11 +14,12 @@ class Phantomjs extends EventEmitter
 
   childProcess: null
 
-  run: (url, options = '--load-images=no --ssl-protocol=TLSv1', script = "phantomjs-test-in-console")=>
+  run: (url, options = '--load-images=no --ssl-protocol=TLSv1', script = "phantomjs-test-in-console", xunitOutput = '')=>
```

Comment:
Phatnomjs is a generic class that shouldn't be aware of xunit output. I'd rather change this from xunitOutput to a spawnArgs array